recipes/lzstring/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
It was meant to be line 1. So thanks for fixing.

Note: Yeah GitHub PR diffs can be pretty dumb like that. I think it searches for a match in the file that overlaps with the snippet from the comment. Probably should have put it on line 2. Then again GitHub should be way smarter about this.